src/server.c

```diff
@@ -1126,6 +1110,40 @@ void clientsCron(void) {
     }
 }
 
+/* A periodic timer that performs client maintenance.
+ * This cron task follows the following rules:
+ *  - All clients need to be checked (at least) once per second
+ *  - To manage latency, we don't check more than MAX_CLIENTS_PER_CLOCK_TICK at a time
+ *  - The minimum rate will be defined by server.hz
+ *  - At least CLIENTS_CRON_MIN_ITERATIONS will be performed each cycle
+ */
+#define CLIENTS_CRON_MIN_ITERATIONS 5
```

Comment:
it's better to align the name with `MAX_CLIENTS_PER_CLOCK_TICK` and put it to `server.h` together, we have the chance now

Reply:
if we wanna minimize the scope, we can put `MAX_CLIENTS_PER_CLOCK_TICK` here and change `CLIENTS_CRON_MIN_ITERATIONS` to `MIN_CLIENTS_PER_CLOCK_TICK`

Reply:
I've aligned the names as you've suggested and localized them both to the function where they are used.